### DISCUSSION TITLE
Form submission with only changed inputs

### DISCUSSION THREAD

**Discussion by Doma1204** [formkit/formkit#939] [General]
The Formkit library has a `context.state.dirty` field for each input, but when the form is submitted, it will return all the inputs of the form. Is there any way that I can receive only the changed inputs when submitting?

**Comment by justin-schroeder** [comment]
@Doma1204 — great question — nuanced behavioral changes like this are where plugins shine. Here’s an example where I used core node hooks (the submit hook) to filter out any values that were no dirty: https://formkit.link/6b0252bb3de6c0e9a5cbee17e53c49af
